app/assets/stylesheets/pages/profiles/_about.scss

```diff
@@ -160,7 +160,26 @@
 //--------------Mobile View--------------------------------
 //---------------------------------------------------------
 @media screen and (max-width: 780px){
-  #main-nav{ display: none; }
+  #main-nav{  }
```

Comment:
reccomend adding an annotation to any dead styles like `` // main navigation which we might want to show/hide at a later time`